Makefile

```diff
@@ -69,7 +76,7 @@ VB_FILE =
 VB_FUNC =
 
 
-.PHONY: all clean test gotest server dev benchkv benchraw check checklist parser tidy ddltest
+.PHONY: all clean test gotest server dev benchkv benchraw check checklist parser tidy ddltest server_global_kill_test
```

Comment:
Could this be put into a standalone Makefile in  tests/globalkilltest/Makefile?

Reply:
Comment addressed, PTAL~ @SunRunAway 

To reuse base Makefile but avoid conflicts of target, I split original Makefile to Makefile.common (for environment variables) & Makefile (for targets).